contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What is a growth engineer (and how they help your startup)
+date: 2023-04-19
+author:
+    - ian-vanagas
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Explainers
+---
+
+Every startup loves to see signups, subscriptions, and revenue go up, yet all engineers basically work on things indirectly related to doing so. The power of growth engineers is that they focus on making these numbers go up, working on whatever it takes.
+
+A growth engineer is an engineer who works on projects impacting the core business metrics, no matter where they are in the product. This is different than regular engineers who work on specific products and features to improve metrics for them. This indirectly improves the core business metrics.
+
+Growth engineers help companies grow faster by focusing on projects with high potential to impact core business metrics. We know this because we have a team of growth engineers at PostHog, and they’ve made a big impact on signups and revenue. 
+
+To help you understand growth engineers better, this post will go over what we’ve learned about the type of people who succeed as growth engineers, what skills they have, what they work on, and why they matter.
+
+## What are the characteristics of a growth engineer?
+
+To start to understand growth engineers, it is useful to understand who they are as people. When comparing them to regular engineers, here are the characteristics that stand out:
+
+![Characteristics](../images/blog/what-is-a-growth-engineer/characteristics.png)
+
+1. **Entrepreneurial**. Growth engineers have an interest in the business as a whole as well as the hustle to find ways to improve it. They find areas they can make a large impact on the core business metrics, and develop solutions themselves to improve those metrics. Like an entrepreneur, they are independent and scrappy.
```

Comment:
Im not sure this is correct. I knew plenty of folks who worked on Growth but werent Entrepreneurial :) So I certainly wouldnt put it first.

Also, another point would be "user/product driven" i.e. need to be passionate about understanding and building great user experiences